src/components/Movies.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./Movies.css";
+
+class Movies extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      selectedOption: "",
+      movie: {},
+    };
```

Comment:
```suggestion
    this.state = {
      movies: [],
      movie: {},
    };
```
The `selectedOption` piece of state is redundant. The only place you're actually using it is to set a "value" on the `select` element, but `select` elements actually do **not** have their own explicit `value` attribute. `e.target.`**`value`** refers to the `value` of the `option` that is currently chosen from the `select` element.
**documentation for `select`, take note of its list of attributes:** https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select
**documentation for `option`, take note of its list of attributes:** https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option